Powered by footer

pranxv opened the pull request:
## Related Issue 

- Info about the related issue 

Closes: #86 
### Describe the changes you've made

Give a clear description what modifications you have made

In the footer section of 'Index.html', added a new <p> class ="footer__credits" with text "Powered by" followed by logos of github and Dev protocol<img> with class = "git__link".
 In style.css, accessed the class "footer_credits" and set the display to inline-flex, and justify content: center and margin of 30px.

## Type of change

What sort of change have you made:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made

## Checklist:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [] I have commented my code, particularly whereever it was hard to understand.
- [] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [] I have added tests that prove my fix is effective or that my feature works.
- [] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |
![Screenshot (241)](https://user-images.githubusercontent.com/40822047/130248687-32a6051a-e562-4af1-ad6c-5f45676eb286.png) | ![Screenshot (239)](https://user-images.githubusercontent.com/40822047/130248679-e149ceec-23fd-46d8-9868-09394bc084ba.png)

vinzvinci reviewed (changes requested):
- [ ] It will be cool if you bring down the logo
- [ ] add GitHub & Dev Protocol text along with their logo
- [ ] insert their website link
  - http://github.com/
  - https://devprotocol.xyz/

pranxv commented:
Have updated with the required changes, please check
Added links along with the images and are aligned center. Added new classes "list__link" as well as "pre__list".

vinzvinci reviewed (approved):
LGTM 👍
